### DISCUSSION TITLE
怎么使用代码的折叠功能

### DISCUSSION THREAD

**Discussion by lyf110** [vuepress-theme-hope/vuepress-theme-hope#3037] [Q&A] [answered]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

如下图所示，我想要使用此按钮，点击之后可以收缩和打开代码框

![image](https://user-images.githubusercontent.com/42530465/232743696-ea530802-ca34-4e28-a652-2eeeef4a5981.png)

![image](https://user-images.githubusercontent.com/42530465/232743994-caf30180-3931-45ae-902f-f4fe5caeaf8e.png)

**Comment by Mister-Hope** [comment]
Issue panel is not the place where you can ask noob questions, for things like this, aheading to discussion panel is welcomed.

**Comment by Mister-Hope** [comment, marked as answer]
Read the docs carefully plz: https://theme-hope.vuejs.press/zh/guide/markdown/container.html

```md
::: details

content here

:::
```
